internal/web/templates/pages/package.html

```diff
@@ -73,8 +73,17 @@ <h2 class="text-reset">Package Details</h2>
               {{ .Manifest.LongDescription }}
             {{ end }}
           </span>
+          <br />
+          <span>
+            {{ if IsUpgradable .Package.Status.Version .LatestVersion }}
+              <i> Latest Version: {{ .LatestVersion }} </i>
+            {{ end }}
+            <br />
+            {{ if ne .Status nil }}
+              <i> Current Version: {{ .Package.Status.Version }} </i>
+            {{ end }}
+          </span>
```

Comment:
I think it would be preferable to use block elements instead of forced line breaks. Something like this:

```suggestion
          {{ if IsUpgradable .Package.Status.Version .LatestVersion }}
            <div>
              <i>Latest Version: {{ .LatestVersion }}</i>
            </div>
          {{ end }}
          {{ if ne .Status nil }}
            <div>
              <i>Current Version: {{ .Package.Status.Version }}</i>
            </div>
          {{ end }}
```

Reply:
Ok will do the necessary changes.